README.md

```diff
@@ -23,165 +23,130 @@ Start Alfresco Content Services  using docker-compose or Kubernetes, containing:
 ```
-
 This will download a linux iso and install it in your Hyper V Manager - you should see a **minikube** VM, after it is installed. It also installs all the needed software in that VM, to simulate a kubernetes cluster.  
-**Note:** you may need to add ```--extra-config=kubelet.ImagePullProgressDeadline=30m0s``` parameter to your start command, as the docker images are rather big.  
+You may need to add ```--extra-config=kubelet.ImagePullProgressDeadline=30m0s``` parameter to your start command, as the docker images are rather big.  
 
-##### helm
-Run this command to initialize helm:
-```bash
-helm init
-```
-Note: If you are using minikube 0.26.0+ (with RBAC) use the following:
+#### Initialize Helm
 ```bash
 kubectl create serviceaccount --namespace kube-system tiller
 kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
 helm init --service-account tiller
 ```
-
-##### commands to check the minikube kubernetes cluster is running:
+#### Add the Alfresco incubator Helm repository
+Add a new repository
 ```bash
-minikube version
-minikube ip
-# the following command opens a web UI of the minikube cluster
-minikube dashboard
-
-helm version
-helm repo list
-
-kubectl version
-kubectl get pods
-kubectl get secrets
+helm repo add alfresco-incubator http://kubernetes-charts.alfresco.com/incubator
 ```
-
-#### Step 2: Install the ingress service:
+Update repository indexes
 ```bash
 helm repo update
-helm install stable/nginx-ingress --version 0.14.0
-# Add '--set rbac.create=true' if the environment has RBAC enabled
-# After running this command look for the name of the ingress deployment.
-# It is usually an animal name preceded by an adjective like: "singed-chipmunk"
-
-# Next get the ip and port of the ingress controller
-minikube ip
-# It will print something like: 172.31.147.123
-
-# To get the port, replace "singed-chipmunk" with the name of
-# your nginx-ingress-controller deployment
-kubectl get service singed-chipmunk-nginx-ingress-controller -o jsonpath={.spec.ports[0].nodePort}
-# This will print a port like: 30917
-
-# Remember the ip from the prev command and port to get the
-# external address value: http://172.31.147.123:30917 that we will use later
 ```
-
-#### Step 3 Add the alfresco incubator helm repository
-
-We need this to add the persistent volume claim defined in the infrastructure. Everything else in the infrastructure is disabled for the moment, but the idea is that in the future we could easily enable components based on the new features that we will enable/implement.
-
-### Add the incubator helm repo:
+Check that the repository was added
 ```bash
-helm repo add alfresco-incubator http://kubernetes-charts.alfresco.com/incubator
 helm repo list
 ```
+#### Install the ingress service:
+Deploy nginx:
+```bash
+helm install stable/nginx-ingress --set rbac.create=true
```

Comment:
Would be better if you pinpoint a version for the ingress here.

Reply:
That's a valid concern, but I think we agreed that it should work with latest (unless there is a bug in ingress).
@derekhulley Can you please confirm? Otherwise I can put the latest chart version that is currently "tested" (0.18.0).

Reply:
The rate of change of the software means that the fixed version will fall behind virtually immediately.
We want our master branch to be current and therefore the instructions should reflect support for the latest versions of the software.  We might pin things once we have support branches.

Reply:
It also goes without saying that we will review any failures that are highlighted as a result of this policy and fix our charts accordingly.